source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,286 @@
+   :copyable:
+   :dedent:
+
+Literal Values
+--------------
+
+Literal value queries return documents with an exact match to your search criteria.
+
+The following example specifies a query as a parameter to the ``Find()`` method,
+using the ``Guitar`` class. The query matches all documents where the ``make`` 
+field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: ../../../includes/fundamentals/code-examples/specify-query/FindEqPOCO.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: none
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+The following example creates a query filter using builders that matches all 
+documents where the ``make`` field equals "Fender".
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: ../../../includes/fundamentals/code-examples/specify-query/FindEqBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: none
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Comparison
+----------
+
+Comparison operators analyze the value in a document against the specified value
+in your match criteria. Common comparison operators include: ``>`` or ``Gt()`` 
+for "greater than", ``<=`` or ``Lte()`` for "less than or equal to", and ``!=``
+or ``Ne()`` for "not equal to".
+
+For a full list of comparison operators, see the :manual:`Comparison
+Query Operators </reference/operator/query-comparison/>` page.
+
+The following example specifies a query as a parameter to the ``Find()`` method,
+using the ``Guitar`` class. The query matches all documents where the ``establishedYear`` 
+field is greater than ``1985``.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: ../../../includes/fundamentals/code-examples/specify-query/FindGtPOCO.cs
+      :language:  csharp
+
+   .. output::
+      :language: none
+      :visible:
+
+      { "_id" : 4, "make" : "Kiesel", "models" : ["Ares", "Vader", "Solo"], "establishedYear" : 2015, "rating" : null }
+
+The following example creates a query filter using builders that matches all 
+documents where the ``establishedYear`` field is greater than ``1985``.
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: ../../../includes/fundamentals/code-examples/specify-query/FindGtBuilder.cs
+      :language:  csharp
+
+   .. output:: 
+      :language: none
+      :visible:
+
+      { "_id" : 1, "make" : "Fender", "models" : ["Stratocaster", "Telecaster"], "establishedYear" : 1946, "rating" : 9 }
+
+
+To learn more about using builders, see :ref:`csharp-builders`.
+
+Logical Operators
+-----------------
+
+Logical operators query data using logic applied to the results of field-level
+operators. Common logical operators include ``&&`` or ``And()`` where all operators must be true, 
```

Comment:
Question/Issue: What is a "field-level operator"? Could this term be defined on the page? I believe this is the only time it is used.

Reply:
Updated to be easier to understand and use less "technical jargon"